src/lib/Microsoft.Health.Events/EventConsumers/Service/EventConsumerService.cs

```diff
@@ -42,6 +42,7 @@ public async Task ConsumeEvents(IEnumerable<IEventMessage> events)
                     catch (Exception e)
                     {
                         _logger.LogError(e);
+                        throw e;
```

Comment:
Should use throw instead of throw e to preserve original stack trace.

Reply:
Also could you be clearer on what bug you saw with this?  For the FHIR conversion we intentially do not throw exceptions because we don't want to loose data for intermittant errors.  Instead errors that are non recoverable (i.e. bad data in the pipeline) we catch and emit telemetry and all others are considered retryable.  I want to make sure we don't lose that feature.